c2rust-refactor/src/scripting.rs

```diff
@@ -587,5 +843,18 @@ impl<'a, 'tcx> UserData for TransformCtxt<'a, 'tcx> {
         methods.add_method("get_ast", |lua_ctx, this, node: LuaAstNode| {
             this.get_lua_ast(lua_ctx, node)
         });
+
+        /// Visits an entire crate via a lua object's methods
+        // @function run_visitor
+        // @tparam function() callback Function called for each function like item.
+        methods.add_method_mut("run_visitor", |lua_ctx, this, (visitor_obj, krate): (LuaTable, LuaAstNode)| {
```

Comment:
I would expect this to be called `visit_crate` as it is analogous to `LuaAstVisitor::visit_crate`.